data/editor.config

```diff
@@ -621,10 +627,16 @@
         <include group="poi" />
         <include field="internet" />
       </type>
+      <type id="shop-chocolate" group="shop">
+        <include group="poi" />
+      </type>
       <type id="shop-clothes" group="shop">
         <include group="poi" />
         <include field="internet" />
       </type>
+      <type id="shop-coffee" group="shop">
+        <include group="poi" />
```

Comment:
Why are you not adding an internet field? Btw, should it be included in POI by default? What are the exceptions without Internet?